### DISCUSSION TITLE
Fixed plasma protein scale factor searches are not available when create population.

### DISCUSSION THREAD

**Discussion by Chelton919** [Open-Systems-Pharmacology/Forum#1602] [Questions & Problems]
Hello everyone.
I need to fix plasma protein scale factor when create population. But I can't find it in the "User Defined Variability”. I have seen before that it is allowed to be searched. It is shown in the picture below. Please tell me the reason, thank you very much.

Chelton.
![1693023862528](https://github.com/Open-Systems-Pharmacology/Forum/assets/116017874/d6420a20-2833-438d-8d85-705f099a0504)
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/116017874/b3ef8dce-58a3-4398-88f2-147639160729)

**Comment by Yuri05** [comment]
cannot reproduce.
Can you upload your project?

**Reply by SCZ1174** [reply]
I found that if the population created is based on "CKD" individual, it cannot modify the "plasma protein scale factor" in the population. If it is a healthy population, it is OK. I have attached a file.
Is this reasonable?
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/86552551/155dde5e-40b2-4f8d-abcb-3db8dd6b33e6)
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/86552551/4120009b-9bca-4794-b74a-7edbacdfa77b)
[plasma protein scale factor-CKD.json](https://github.com/Open-Systems-Pharmacology/Forum/files/14766536/plasma.protein.scale.factor-CKD.json)
